app/controllers/api/functions.php

```diff
@@ -1114,3 +1114,201 @@
 
         $response->noContent();
     });
+
+// Variables
+
+App::post('/v1/functions/variables/:functionId')
+    ->desc('Create Variable')
+    ->groups(['api', 'functions'])
```

Comment:
We miss the `sdk.description` label with a link to a markdown file on all new endpoints.